activities/TankOp.activity/locale.ini

```diff
@@ -95,6 +127,22 @@ Joshfeed=joshfeed from freesound (raté)
 Danipenet=DANIpeNet from freesound (mission terminée)
 Juskiddink=juskiddink from freesound (mission echouée)
 
+TutoPrev=Préc
+TutoNext=Suiv
+TutoEnd=Fin
+TutoExplainTitle=Activité d'exploitation de chars
```

Comment:
Don't do automatic translation, you could use the english localization for all languages

Reply:
@llaske Should I use english localization for all titles or should I just use it for only these 4?

Reply:
Use it for all. Native speakers will translate them

Reply:
@llaske automatically translated text replaced with english.

@sashr - Thank you for clarification.
> Use it for all. Native speakers will translate them